fixtures/dom/src/components/fixtures/date-inputs/switch-date-test-case.js

```diff
@@ -0,0 +1,54 @@
+const React = window.React;
+
+/**
+ * This test case was originally provided by @richsoni,
+ * https://github.com/facebook/react/issues/8116
+ */
+class SwitchDateTestCase extends React.Component {
+  state = {
+    fullDate: false,
+    date: new Date().toISOString()
+  }
+
+  render() {
+    const attrs = this.inputAttrs();
+
+    return (
+      <div>
+        <p><b>{attrs.type}</b> input type ({attrs.value})</p>
+        <p>
+          <input type={attrs.type} value={attrs.value} onChange={this.onInputChange} />
+          <label><input type='checkbox' checked={this.state.fullDate} onChange={this.updateFullDate} /> Switch type</label>
+        </p>
+      </div>
+    );
+  }
+
+  inputAttrs() {
+    if (this.state.fullDate){
```

Comment:
Ah man - you know what? Prettier/Lint doesn't run on the fixtures!

Otherwise I use prettier on every single project :)